### ISSUE TITLE
Client group checklist visibility

### ISSUE THREAD

**Issue by markstreaka** [streaka/plantguard-issues#280]
As a client I want to be able to determine which client group checklists are visible to suppliers.
Using Aurizon as an example:
"Equipment Hire Pre Delivery Checklist" and "Restart Checklist for Plant Fitted with Height Limiters" are checklists that can be completed by supplier employees and Aurizon employees and should therefore be visible to the Suppliers and the Client. 
Whereas:
"Raise Plant Issues Form" should only be visible to the client (Aurizon) as it will only be completed by Aurizon employees.
So to summarise in the checklists builder when a Client is creating a checklist they should be able to toggle yes/no for:
Client Group Checklist
then if yes, toggle yes/no for:
Visible to Suppliers 

UNLESS of course if you suggest something different

**Comment by markstreaka** [comment]
When I review the video from Friday (http://streaka.testing.videos.s3.amazonaws.com/screencast_2017-06-16_16-08-58.mp4) - I think just completing this issue should fix all the problems I outlined there.

**Comment by alexkingma1** [comment]
- [ ] denote private/public nature of checklists on each checklist template table
- [ ] extra select/toggle field in builder to initially set this private/public value
- [ ] double-check existing checklist visibilities
